src/panels/lovelace/editor/card-editor/hui-dialog-edit-card.ts

```diff
@@ -402,6 +404,9 @@ export class HuiDialogEditCard extends LitElement implements HassDialog {
           ha-dialog {
             --mdc-dialog-max-width: calc(100% - 32px);
             --mdc-dialog-min-width: 1000px;
+            --dialog-surface-position: fixed;
+            --dialog-surface-top: 40px;
+            --mdc-dialog-max-height: calc(100% - 72px);
```

Comment:
This is to prevent it from jumping? Does it also work on mobile?

Reply:
This isn't applied on mobile. mobile is full screen